internal/dockerutil/setup.go

```diff
@@ -89,8 +91,9 @@ func DockerSetup(t DockerSetupTestingT) (*client.Client, string) {
 // dockerCleanup will clean up Docker containers, networks, and the other various config files generated in testing
 func dockerCleanup(t DockerSetupTestingT, cli *client.Client) func() {
 	return func() {
+		showContainerLogs := os.Getenv("SHOW_CONTAINER_LOGS")
```

Comment:
Instead of a later check that `showContainerLogs != ""`, can you set this as a bool, i.e. `showContainerLogs := os.Getenv("SHOW_CONTAINER_LOGS") != ""`?

Reply:
done